wrangler-transform/src/e2e-test/java/io/cdap/plugin/wrangler/runners/TestRunner.java

```diff
@@ -26,7 +26,7 @@
 @CucumberOptions(
   features = {"src/e2e-test/features"},
   glue = {"stepsdesign", "io.cdap.plugin.common.stepsdesign", "io.cdap.plugin.wrangler.stepsdesign",
-          "io.cdap.plugin.wrangler.locators"},
+          "io.cdap.plugin.wrangler.actions"},
```

Comment:
why do we need to add both "io.cdap.plugin.wrangler.stepsdesign" and "io.cdap.plugin.wrangler.actions" packages in the glue ? Shouldn't the step definitions for wrangler reside in one package ?

Reply:
io.cdap.plugin.wrangler.stepsdesign is for wrangler class to validate the data and io.cdap.plugin.wrangler.actions it contains wrangler actions so that's why we put them in different packages.we have a common folder named **wrangler** in which these two folders exists.